public/app/themes/justice/src/sass/admin.scss

```diff
@@ -0,0 +1,33 @@
+.toplevel_page_revisionary-q {
+    a[href="admin.php?page=revisionary-pro"] {
```

Comment:
I think targeting the href is the best way to do this.

Can you check it works locally?

Reply:
Yes, this works locally.